client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -330,6 +330,8 @@ function useShowSiteCreationNotice( allSites: SiteExcerptData[], newSiteSlug: st
 			return;
 		}
 
+		site.is_creating = true;
```

Comment:
> @javierarce could you help us here? I feel a better option would be to use the "Plan expired" banner inside the site to indicate processing

![image](https://github.com/Automattic/wp-calypso/assets/47489215/42e4eae2-e9cf-49e2-8a0e-f1aef22206a9)

Think showing banner like this over the thumbnail like the expired plan make sense, just to change the wording.

Reply:
Could we show the loading ellipsis and a title instead? This would look nicer than using the banner and we would be using the same loading style as in the onboardings. 

![image](https://github.com/Automattic/wp-calypso/assets/4933/ac50b3bc-4de6-40c0-b1b2-a256b86441c5)

Reply:
Yes @vykes-mac

@javierarce I was thinking of maintaining consistency. I don't like the idea of modifying the thumbnail 🤔 Adding the fixed banner on top of the tile image makes more sense to me, but you have the final saying 😁 

Like this:

![image](https://github.com/Automattic/wp-calypso/assets/26530524/931df575-da5e-46f8-8ff9-df0e43252697)

But instead of error, it would be an info banner with a spinner. On a white background.

Reply:
Something like this
https://github.com/Automattic/wp-calypso/assets/47489215/cd6b154f-7ced-4c7c-9fd7-4e6e345370fc

Reply:
That looks great @vykes-mac

Is there any way to dim the card slightly so that it shows up as disabled?

Reply:
Well, thinking better, maybe it's not a good idea since the user can already see the site through `/home/%s`, so let's not dim it.

Reply:
Ok, for now, let's use @vykes-mac's proposed solution to keep consistency with the expired notice.